laser_filter:
  ros__parameters:
    filter1:
      name: mask
      type: laser_filters/LaserScanMaskFilter
      params:
        masks:
          laser:
#          - 43.
          - 44.
          - 45.
          - 46.
          - 47.
          - 48.
          - 49.
          - 50.
          - 51.
          - 52.          
          - 53.
          - 54.          
          - 55.
          - 56.
          - 57.
# ..............          
          - 173.
          - 174.
          - 175.
          - 176.
          - 177.
          - 178.
          - 179.
          - 180.
          - 181.
          - 182.
          - 183.
          - 184.
          - 185.          
          - 186.
          - 187.
          - 188.
# ..............         
          - 519.  
          - 520.
          - 521.
          - 522.
          - 523.
          - 524.
          - 525.
          - 526.          
          - 527.
          - 528.
          - 529.
          - 530.
          - 531.
          - 532.
          - 533.
          - 534.
          - 535.
          - 536.
# ..............
          - 642.          
          - 643.
          - 644.
          - 645.
          - 646.
# ..............                    
          - 654.                   
          - 655.          
          - 656.
          - 657.
          - 658.
          - 659.
          - 660.
          - 661.
          - 662.
          - 663.
          - 664.
          - 665.